sdk/storage/storage-blob/src/BlobClient.ts

```diff
@@ -1284,6 +1293,28 @@ export class BlobClient extends StorageClient {
     }
   }
 
+  public async beginCopyFromURL(copySource: string, options: BlobBeginCopyFromURLOptions = {}) {
```

Comment:
>beginCopyFromURL [](start = 15, length = 16)

need to discuss the naming & interface. There is already a `startCopyFromURL`

Reply:
@bterlson Is `beginCopyFromURL` suppose to replace `startCopyFromURL` ?

Reply:
I went with beginCopyFromURL to match the pattern in keyvaults. Also want to know if there’s value in having the poller version and the non-poller version.

Reply:
Confirmed with @bterlson  that this method should be `beginCopyFromURL`, and `startCopyFromURL` should not be public.